srcpkgs/gcr4/template

```diff
@@ -0,0 +1,48 @@
+# Template file for 'gcr4'
+pkgname=gcr4
+version=4.0.0
+revision=1
+build_style=meson
+build_helper="gir"
+configure_args="$(vopt_bool gir introspection) $(vopt_bool gir gtk_doc)"
+hostmakedepends="gettext glib-devel gnupg gi-docgen gettext pkg-config openssh
+ libxslt $(vopt_if gir vala)"
+makedepends="libgcrypt-devel libsecret-devel p11-kit-devel libxslt-devel gtk4-devel"
+depends="desktop-file-utils hicolor-icon-theme gnupg"
+checkdepends="dbus"
+short_desc="GNOME crypto package"
+maintainer="Orphaned <orphan@voidlinux.org>"
+license="LGPL-2.0-or-later, LGPL-2.1-or-later, GPL-2.0-or-later"
+homepage="https://gitlab.gnome.org/GNOME/gcr"
+changelog="https://gitlab.gnome.org/GNOME/gcr/-/raw/master/NEWS"
+distfiles="${GNOME_SITE}/gcr/${version%.*}/gcr-${version}.tar.xz"
+checksum=c45855924f0ee7bab43e2dd38bfafd2ac815c6e9864341c0161e171173dcec7c
+make_check_pre="dbus-run-session"
+
+# Package build options
+build_options="gir"
+build_options_default="gir"
+
+post_patch() {
+	vsed -e "s/gcr_soversion = .*/gcr_soversion = '0'/" -i meson.build
+	vsed -e "s/gck_soversion = .*/gck_soversion = '0'/" -i meson.build
+}
```

Comment:
this was just for testing, it can e dropped

Reply:
Alright.